model/src/main/resources/db-update/00006-from-1.5.1-to-1.6.0.sql

```diff
@@ -0,0 +1,27 @@
+--
+-- JBoss, Home of Professional Open Source.
+-- Copyright 2014 Red Hat, Inc., and individual contributors
+-- as indicated by the @author tags.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+
+-- [NCL-4652]
+-- Copy scmRevision to scmTag for old builds
+alter table buildrecord add column scmTag varchar(255);
+
+UPDATE buildrecord
+SET scmTag = subquery.scmRevision
```

Comment:
This seems very strange to me. Couldn't this be just
```UPDATE buildrecord SET scmTag = scmRevision;```
It assigns the scmRevision value to scmTag from the same row. Or wouldn't that worked for some reason?

Reply:
Honestly, I have never tried such update. I can test it out.

Reply:
You were right. It was enough. Simplified. Merging